wasp/src/main/java/com/orhanobut/wasp/toolbox/WaspBitmapCache.java

```diff
@@ -0,0 +1,22 @@
+package com.orhanobut.wasp.toolbox;
+
+import android.graphics.Bitmap;
+
+import com.android.volley.toolbox.ImageLoader;
+import com.orhanobut.wasp.WaspCache;
+
+/**
+ * @author Orhan Obut
+ */
+public class WaspBitmapCache extends WaspCache<String, Bitmap> implements ImageLoader.ImageCache {
+
```

Comment:
I agree, we need to provide more custom feature. We should add this to todo and provide a good solution via Wasp.Builder